src/Microsoft.Health.Dicom.Core/Exceptions/Validation/DateIsInvalidException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Dicom;
+using Microsoft.Health.Dicom.Core.Features.Validation;
+
+namespace Microsoft.Health.Dicom.Core.Exceptions.Validation
+{
+    public class DateIsInvalidException : ElementValidationException
```

Comment:
Not sure if good idea to create Exception for each error code. The intention is to organize error code and associated error message together.

Reply:
I think single `ElementValidationException` might be a cleaner solution. 
I created several static methods to generate exceptions for different error code, and encapsulate them into static class. in this way, don't need so many classes, also maintain the connection between error code and message, so that can easily create `ElementValidationException` for specific error code.